mad-dog/src/tests/test_userReplicaSetManager.js

```diff
@@ -230,8 +255,29 @@ const verifyUrsmContentNodes = async (executeOne) => {
         )
       }
       logger.info(`Found UserReplicaSetManager and ServiceProviderFactory match for spID=${spID}, delegateWallet=${delegateWalletFromChain}, ownerWallet=${ownerWalletFromChain}`)
+
+      /**
+       * Confirm nodes with spIDs 1-3 are correctly configured as bootstrappers and any additional nodes were registered with proposals from registered nodes
```

Comment:
just to clarify, we know the order here because ursm content nodes are ordered by ID in the route from disc prov and the requesting node aggregates the proposer responses in said order?

Reply:
nah we don't know the order in which the requesting node aggregates proposer responses. this code works bc it makes no assumptions on the order of those responses